5001-6000/LIT5051MenbabFelsata.xml

```diff
@@ -48,6 +57,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
         </profileDesc>
         <revisionDesc>
             <change who="DR" when="2018-08-20">Created entity</change>
+            <change who="MKr" when="2024-07-12" >Added clavis and abstract</change>
```

Comment:
I think, you need to remove the space inside` <change >` (`<change who="MKr" when="2024-07-12 >`). Otherwise, it will probably cause problems in the app.